solutions/src/main/java/ru/sberbank/school/task02/validators/AmountPositiveValidator.java

```diff
@@ -0,0 +1,17 @@
+package ru.sberbank.school.task02.validators;
+
+import java.math.BigDecimal;
+
+/**
+ * Валидирует положительное значение входящей суммы
+ * Если сумма не положительная кидает IllegalArgumentException
+ * Created by Gregory Melnikov at 11.05.2019
+ */
+public class AmountPositiveValidator {
+
+    public static void amountValidate (BigDecimal amount) {
```

Comment:
&#39;(&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1148928319>*